tests/test_bgp_af.rb

```diff
@@ -343,10 +533,17 @@ def test_additional_paths_selection
       bgp_af.default_additional_paths_selection
 
     af_string = get_bgp_af_cfg(asn, vrf, af)
+    if platform == :nexus
+      af_string = get_bgp_af_cfg(asn, vrf, af)
+    elsif platform == :ios_xr
+      af_string = @device.cmd("show run router bgp #{asn} vrf #{vrf} " \
+                              'address-family ipv4 unicast')
```

Comment:
I think my comment got lost in the shuffle - I would much prefer if you refactor `get_bgp_af_cfg` to work for both nexus and ios_xr: `@device.cmd('show run router bgp #{asn} vrf #{vrf} address-family #{afi} #{safi}')`